moveit_core/robot_state/include/moveit/robot_state/robot_state.h

```diff
@@ -1151,31 +1185,47 @@ as the new values that correspond to the group */
                               const GroupStateValidityCallbackFn& validCallback = GroupStateValidityCallbackFn(),
                               const kinematics::KinematicsQueryOptions& options = kinematics::KinematicsQueryOptions())
   {
-    return computeCartesianPath(group, traj, link, waypoints, global_reference_frame, max_step,
+    return computeCartesianPath(group, traj, link, waypoints, global_reference_frame, MaxEEFStep(max_step),
                                 JumpThreshold(jump_threshold_factor), validCallback, options);
   }
 
+  /** \brief Helper function that calls both testJointSpaceJump and testCartesianSpaceJump
+  */
+  static double trimJointAndCartesianSpaceJumps(const JointModelGroup* group, const LinkModel* link,
```

Comment:
why introduce this term "trim"? the trajectories are not actually shortened are they? only the 'percent' changes. since they call "test" functions internally then this should also be "testJointAndCartesianSpaceJumps"... although in my other comment i explained why i think we should rename test to "check"